.github/workflows/datasets-batch-deployer.yml

```diff
@@ -75,14 +75,45 @@ jobs:
           scripts/replace-variables.sh -in_file infra/batch/vars.tfvars.rename_me -out_file infra/batch/vars.tfvars -variables REGION,PROJECT_ID,DEPLOYER_SERVICE_ACCOUNT,JOB_SCHEDULE,ENVIRONMENT,DATASETS_BUCKET_NAME
           cat infra/batch/vars.tfvars
 
-      - name: Docker Compose DB/Liquibase for db-gen.sh
-        run: docker-compose --env-file ./config/.env.local up -d liquibase
+      - name: Docker Compose DB
```

Comment:
We can combine them. The downside is this step here is only generating the models and the _build & test_ execute tests for API and Python functions. We can extract just the code generation, but it might reduce the time and increase cost as done in a different runner. Let's keep it as it is now and refactor if this step becomes more complex.